{
    "CVE-ID": "CVE-2022-48564",
    "NVD": {
        "Descriptions": "read_ints in plistlib.py in Python through 3.9.1 is vulnerable to a potential DoS attack via CPU and RAM exhaustion when processing malformed Apple Property List files in binary format.",
        "Cvss2_BaseScore": 0,
        "Cvss3_BaseScore": 6.5,
        "Cvss2_VectorString": "",
        "Cvss3_VectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H",
        "CPEs": [
            {
                "URI": "cpe:/a:python:python",
                "FormattedString": "cpe:2.3:a:python:python:*:*:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "",
                "VersionEndExcluding": "3.6.13",
                "VersionEndIncluding": ""
            },
            {
                "URI": "cpe:/a:python:python",
                "FormattedString": "cpe:2.3:a:python:python:*:*:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "3.7.0",
                "VersionEndExcluding": "3.7.10",
                "VersionEndIncluding": ""
            },
            {
                "URI": "cpe:/a:python:python",
                "FormattedString": "cpe:2.3:a:python:python:*:*:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "3.8.0",
                "VersionEndExcluding": "3.8.7",
                "VersionEndIncluding": ""
            },
            {
                "URI": "cpe:/a:python:python",
                "FormattedString": "cpe:2.3:a:python:python:*:*:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "3.9.0",
                "VersionEndExcluding": "3.9.1",
                "VersionEndIncluding": ""
            },
            {
                "URI": "cpe:/a:netapp:active_iq_unified_manager:-::~~~windows~~",
                "FormattedString": "cpe:2.3:a:netapp:active_iq_unified_manager:-:*:*:*:*:windows:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "",
                "VersionEndExcluding": "",
                "VersionEndIncluding": ""
            },
            {
                "URI": "cpe:/a:netapp:active_iq_unified_manager:-::~~~vmware_vsphere~~",
                "FormattedString": "cpe:2.3:a:netapp:active_iq_unified_manager:-:*:*:*:*:vmware_vsphere:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "",
                "VersionEndExcluding": "",
                "VersionEndIncluding": ""
            }
        ]
    },
    "Official": {
        "3.6.13": {
            "source": [
                "https://github.com/python/cpython/blob/3.6/Misc/NEWS.d/3.6.13.rst"
            ],
            "BTS": {
                "bpo42103": {
                    "ID": "bpo42103",
                    "url": "https://bugs.python.org/issue42103",
                    "date": "2020-10-23-19-20-14",
                    "nonce": "C5obK2",
                    "section": "Library"
                }
            }
        },
        "3.7.10": {
            "source": [
                "https://github.com/python/cpython/blob/3.7/Misc/NEWS.d/3.7.10.rst"
            ],
            "BTS": {
                "bpo42103": {
                    "ID": "bpo42103",
                    "url": "https://bugs.python.org/issue42103",
                    "date": "2020-10-23-19-20-14",
                    "nonce": "C5obK2",
                    "section": "Library"
                }
            }
        },
        "3.8.7rc1": {
            "source": [
                "https://github.com/python/cpython/blob/v3.8.7rc1/Misc/NEWS.d/3.8.7rc1.rst"
            ],
            "BTS": {
                "bpo42103": {
                    "ID": "bpo42103",
                    "url": "https://bugs.python.org/issue42103",
                    "date": "2020-10-23-19-20-14",
                    "nonce": "C5obK2",
                    "section": "Library"
                }
            }
        },
        "3.9.1rc1": {
            "source": [
                "https://github.com/python/cpython/blob/v3.9.8/Misc/NEWS.d/3.9.1rc1.rst"
            ],
            "BTS": {
                "bpo42103": {
                    "ID": "bpo42103",
                    "url": "https://bugs.python.org/issue42103",
                    "date": "2020-10-23-19-20-14",
                    "nonce": "C5obK2",
                    "section": "Library"
                }
            }
        },
        "3.10.0a2": {
            "source": [
                "https://github.com/python/cpython/blob/v3.14.0a1/Misc/NEWS.d/3.10.0a2.rst"
            ],
            "BTS": {
                "bpo42103": {
                    "ID": "bpo42103",
                    "url": "https://bugs.python.org/issue42103",
                    "date": "2020-10-23-19-20-14",
                    "nonce": "C5obK2",
                    "section": "Library"
                }
            }
        }
    }
}